src/pytorch_lightning/strategies/colossalai.py

```diff
@@ -264,23 +267,54 @@ def setup_precision_plugin(self) -> None:
                 )
         assert isinstance(self.model, (pl.LightningModule, _LightningPrecisionModuleWrapperBase))
         pl_module = self.model
-        process_group = ProcessGroup()
+
+        new_version_flag = compare_version("colossalai", operator.gt, "0.1.10")
+
```

Comment:
Could you define it at the top of the file as

```py
_COLOSSALAI_GREATER_EQUAL_0_1_10 = compare_version("colossalai", operator.gt, "0.1.10")
```

and reuse it here in the code?
We follow this convention across the codebase and it would be nice to keep it standardized.